extern/CMakeLists.txt

```diff
@@ -106,6 +108,7 @@ if(MIOPEN_ENABLE_SQLITE)
     target_link_directories(sqlite3::sqlite3 INTERFACE ${install_dir})
```

Comment:
When you change `INSTALL_COMMAND`, you need to provide new paths here instead of `install_dir`:
`target_link_directories` -> `${CMAKE_BINARY_DIR}\lib`
`target_include_directories` -> `${CMAKE_BINARY_DIR\include`